src/components/organisms/Navbar/MobileDropdown.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { styled } from "@stitches/react";
+import { mauve } from "@radix-ui/colors";
+import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
+import { FiMenu } from "react-icons/fi";
+import { slideDownAndFade, slideLeftAndFade, slideRightAndFade, slideUpAndFade } from "./keyframes/keyframes";
+
+const StyledContent = styled(DropdownMenuPrimitive.Content, {
+  minWidth: 300,
+  backgroundColor: "white",
+  borderRadius: 6,
+  padding: 5,
+  boxShadow: "0px 10px 38px -10px rgba(22, 23, 24, 0.35), 0px 10px 20px -15px rgba(22, 23, 24, 0.2)",
+  "@media (prefers-reduced-motion: no-preference)": {
+    animationDuration: "400ms",
+    animationTimingFunction: "cubic-bezier(0.16, 1, 0.3, 1)",
+    animationFillMode: "forwards",
+    willChange: "transform, opacity",
+    '&[data-state="open"]': {
+      '&[data-side="top"]': { animationName: slideDownAndFade },
+      '&[data-side="right"]': { animationName: slideLeftAndFade },
+      '&[data-side="bottom"]': { animationName: slideUpAndFade },
+      '&[data-side="left"]': { animationName: slideRightAndFade },
+    },
+  },
+});
+
+const itemStyles = {
```

Comment:
use the `css()` function from `@theme`. What's returned can directly be used with the `styled()` function

https://stitches.dev/docs/api#css